docs/daemon/configuration/configuration.mdx

```diff
@@ -0,0 +1,139 @@
+
+## app   
+This section contains configuration specific to the operation of kwild application.
+
+- `grpc_listen_addr`: This variable sets the address at which the gRPC server of the daemon listens for connections. The default value is :50051.
+
+- `http_listen_addr`: This variable sets the address at which the HTTP server of the daemon listens for connections. The default value is :8080.
+
+- `sqlite_file_path`: This variable sets the file path for all SQLite files. If not provided, it defaults to `data/kwil.db/` under the specified root_dir.
+
+- `extension_endpoints`: This variable defines endpoints for extensions. Multiple endpoints can be specified, separated by commas.
+
+- `tls_cert_file`: This variable sets the path to a file containing certificate that is used to create the HTTPS RPC server. The path can be either an absolute path or path relative to the configured home_dir. If not provided, only http server would be created. 
+
+- `tls_key_file`: This variable sets the path to a file containing matching private key that is used to create the HTTPS RPC server. The path can be either an absolute path or path relative to the configured home_dir. If not provided, only http server would be created.
+
+- `hostname`: This variable sets the kwild server hostname.
+
+## chain:
+This section contains configuration specific to cometbft based blockchain.
+
+- `moniker`: This variable sets a custom human-readable name for the node.
+
+### rpc
+This subsection contains the configuration required to manage blockchain's rpc service.
+
+- `laddr`: This variable sets the TCP or UNIX socket address for the RPC server to listen on. Default listening address is on "tcp://127.0.0.1:26657"
+
+- `timeout_broadcast_tx_commit`: This variable sets the time to wait for a transaction to be committed during */broadcast_tx_commit*. Default timeout is 10s.
+
+### consensus
+This subsection describes the configuration to manage consensus.
+
+- `timeout_propose`: This variable sets the time to wait for a proposal block before prevoting nil. Default timeout is 3s.
+
+- `timeout_prevote`: This variable sets the time to wait after receiving +2/3 prevotes for "anything". Default timeout is 1s.
+
+- `timeout_precommit`: This variable sets the time to wait after receiving +2/3 precommits for "anything". Default timeout is 1s.
+
+- `timeout_commit`: This variable sets the time to wait after committing a block before starting on the new height. Default timeout is 1s.
+
+### p2p
+This subsection describes the configuration to manage the p2p service handling peer connectivity between the nodes in the blockchain network.
+
+- `laddr`: This variable sets the Address to listen for incoming connections. Default listening address is "tcp://0.0.0.0:26656".
+
+- `external_address`: This variable sets the Address to advertise to peers for them to dial. If empty, will use the same port as the laddr, and will introspect on the listener.
```

Comment:
Seems like "laddr" should be `'laddr`, or described as the "listening address".  Also, heads up on the grammar here ("If empty, it will use...")